CHANGELOG.md

```diff
@@ -21,7 +21,10 @@ All notable changes to this project will be documented in this file.
 - testing-tools: Add krb5-user library for Kerberos tests ([#531]).
 - testing-tools: Add the Python library Beautiful Soup 4 ([#536]).
 - java-base: Add `openjdk-devel` package for tool such as `jps` or `jmap` ([#537]).
+- java-base: Add JDK 21 ([#547]).
 - opa: Add version `0.61.0` ([#538]).
+- trino: Add version `438` ([#547]).
+- vector: Add version `0.35.0` ([#547]).
```

Comment:
```suggestion
- vector: Switch from version `0.33.0` to `0.35.0` ([#547]).
```

Reply:
ah yeah it is actually a switch (although other images still reference 0.33.0, which i wanted to change in each PR).